Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -363,9 +363,8 @@ instance NeZero.one : NeZero (1 : Ordinal) :=
 `α.toType` into `β.toType`. -/
 def initialSegToType {α β : Ordinal} (h : α ≤ β) :
     @InitialSeg α.toType β.toType (· < ·) (· < ·) := by
-  change α.out.r ≼i β.out.r
-  rw [← Quotient.out_eq α, ← Quotient.out_eq β] at h; revert h
-  cases Quotient.out α; cases Quotient.out β; exact Classical.choice
+  apply Classical.choice (type_le_iff.1 _)
```

Comment:
I personally think using `.mp` or `.mpr` is cleaner than `.1` or `.2`: I will not insist on this, however. Same below.